Add new design styles

Daguilar94 opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe:
```
change styling

## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: #8 


## What is the new behavior?

Before            |  After
:-------------------------:|:-------------------------:
<img width="1440" alt="screen shot 2018-10-06 at 2 38 12 am" src="https://user-images.githubusercontent.com/25178101/46568839-5acfe080-c911-11e8-9c75-9e350bec6470.png">  |  <img width="1440" alt="screen shot 2018-10-06 at 2 37 33 am" src="https://user-images.githubusercontent.com/25178101/46568827-36740400-c911-11e8-963f-913b170a4b05.png">

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

alejandronanez commented on src/Select.js:
```diff
@@ -1,22 +1,28 @@
+  font-family: 'Roboto', sans-serif;
+  font-size: 17px;
+  height: 35px;
+  padding: 10px;
```
I'd remove the padding here. What I understand from this code is that you're using `padding` to separate the `<Select />` component from the container.

alejandronanez commented:
👋 @josemariogutierrez do you have any feedback regarding the color palette @Daguilar94 is using to revamp the look and feel of this playground?

Calling you out since you already created the `favicon` in #31

josemariogutierrez commented:
> 👋 @josemariogutierrez do you have any feedback regarding the color palette @Daguilar94 is using to revamp the look and feel of this playground?
> 
> Calling you out since you already created the `favicon` in #31

Sure, i think that the main color #5c3799 could be preserved to match the color of the React Medellín logo, thats why i used it in the favicon proposal. If you want i can help with the design styles as well.

Daguilar94 commented:
Hi, @alejandronanez , I resolved your comments and switch the main color back to #5c3799 as @josemariogutierrez suggested.

What do you think?

alejandronanez commented:
@daguilar94 first of all, thanks for the flexibility!!!
I’m not really convinced about the black gradient on the left, but I’m not a designer so ¯\_(ツ)_/¯

@josemariogutierrez your call on this one. If you think there’s a better aporoach, feel free to suggest it. This project is yours too 😬

alejandronanez commented:
👋  @josemariogutierrez It's been a while and we know you might have other obligations, but we would to know if you have any input on this?

josemariogutierrez commented:
> 👋 @josemariogutierrez It's been a while and we know you might have other obligations, but we would to know if you have any input on this?

Sure! I also think that the gradient is not very appealing, i can work this weekend on a proposal for the styles :).

josemariogutierrez commented:
Hi @alejandronanez @Daguilar94 , i have made a PR with the design proposal, here is a screenshot:

<img width="932" alt="screen shot 2018-10-14 at 7 43 39 pm" src="https://user-images.githubusercontent.com/27037672/46924664-8a0adf80-cfec-11e8-9f49-d1e614abb154.png">

Sorry if i am not doing the implementation of the styles the way it should, i just started to learn React. I hope you like the proposal.

alejandronanez commented:
@josemariogutierrez that looks **FANTASTIC** 😍, thanks for pulling that off!

@Daguilar94 let me know if you have questions about how to implement this! Also if you think this is too big for one PR, that's fine too!

Just let us know!

josemariogutierrez commented:
> @josemariogutierrez that looks **FANTASTIC** 😍, thanks for pulling that off!
> 
> @Daguilar94 let me know if you have questions about how to implement this! Also if you think this is too big for one PR, that's fine too!
> 
> Just let us know!

@alejandronanez I already implement the design, you can check it in this PR https://github.com/react-medellin/flexbox-playground/pull/35

alejandronanez commented:
> @alejandronanez I already implemented the design, you can check it in this PR #35

Oh, gotcha! Now we have 2 _duplicated_ PRs then! 😅 

Let me think what we can do about this, what you guys suggest? @Daguilar94 @josemariogutierrez  sorry for the confusion here!

josemariogutierrez commented:
> > @alejandronanez I already implemented the design, you can check it in this PR #35
> 
> Oh, gotcha! Now we have 2 _duplicated_ PRs then! 😅
> 
> Let me think what we can do about this, what you guys suggest? @Daguilar94 @josemariogutierrez sorry for the confusion here!

Sorry, my bad, i didn't mean to duplicate the PR, i was trying to learn more of React by applying the styles too 😅.
I can remove the PR and let @Daguilar94 approach the code.

alejandronanez commented:
@josemariogutierrez gotcha! Let's close your PR then so @Daguilar94 can continue with this!

Daguilar94 commented:
@josemariogutierrez  Wow, that looks great!

@alejandronanez Sorry for being away these days, I've been kind of busy but I'll check the code and maybe get this done by tomorrow night.

I'm looking forward to see this new design in master :ok_hand:.

alejandronanez commented:
@daguilar94 no worries at all! We are aware that everybody has obligations :)

Let me know if I can help you with anything, happy to help!

Daguilar94 commented:
Hey, @josemariogutierrez  I think you implemented the styles pretty well, I only changed some things in the FlexboxContainer width because it was breaking when no-wrap option was selected, and also some layout styles. 

Additionally I updated the code to the last commit in master and adjusted the styles of the new components that were added. Let me know what you think about it. @alejandronanez 

This experience helped me a lot to understand styled components, never really worked with that before! 🤓
